src/react/components/window/Flyout.jsx

```diff
@@ -16,7 +16,8 @@ const Container = styled.div`
     min-width: 300px;
     border: 1px solid rgba(0, 0, 0, 0.2);
     max-width: 100vw;
-    max-height: 100vh;
+    max-height: 100%;
```

Comment:
This means that the flyout can be bigger than theres space on the screen and we don't really want that.